index.html

```diff
@@ -194,9 +219,10 @@ <h4>
                 </li>
               </ol>
             </li>
-            <li>If a file type is being blocked due to security considerations,
-            return <a>a promise rejected with</a> with a {{"NotAllowedError"}}
-            {{DOMException}}.
+            <li>If |data|'s {{ShareData/files}} member is present and
```

Comment:
The support check should be split out into its own algorithm (see #177).

Reply:
Note here, we need to do the same check as in #177 to make sure `{files: []}` gets rejected, but passing `{text: "hi", files: []}` is ok.